File: L1.cpp
#include <iostream>
using namespace std;
//Problem 1 of leet code
class Solution{
   public: 
        vector<int> TwoSum(vector<int>& nums, int target){
            int n = nums.size(); //initializing a variable and storing the size of nums
            for(int i=0;i<n-1;i++){
                for(int j= i+1; j<n; j++){
                    if(nums[i]+ nums[j]== target){
                        return{i,j};
                    }
                }
            }
            return {};
        }
};

File: LeetCode/L1.java
//Problem 1 of leet code
class Solution{
    public int[] TwoSum(int[] nums, int target){
        for(int i=0; i<nums.length-1;i++){ //Two loops
            //The first loop will iterate through the array
            //The second loop will iterate through the array starting from the next index of the first loop
            for(int j=i+1;j<nums.length;j++){
                if(nums[i]+nums[j]==target){
                    return new int[] {i,j};
                }
            }
        }
            return new int[] {};  
    }
}
/*
 * A even faster solution
 * class Solution {
    public int[] twoSum(int[] nums, int target) {
        for (int i=1; i<nums.length; i++)
            for(int j=i;j<nums.length; j++)
                if((nums[j]+nums[j-i]) == target)
                    return new int[]{j,j-i};
        return null; 
    }
        
}
 */